1-js/01-getting-started/3-code-editors/article.md

```diff
@@ -1,46 +1,46 @@
-# Code editors
+# குறிமுறை தொகுப்பி
 
-A code editor is the place where programmers spend most of their time.
+ஒரு நிரலர்(programmer) தன்னுடைய அதிக நேரத்தை செலவிடும் இடம் குறிமுறை தொகுப்பி ஆகும்.
 
-There are two main types of code editors: IDEs and lightweight editors. Many people use one tool of each type.
+குறிமுறை தொகுப்பியில் இரண்டு முக்கிய வகைகள் உள்ளன: ஐடிஇ(IDE)க்கள் மற்றும் இலகுரக தொகுப்பிகள். ஒவ்வொரு வகையிலும் பலர் ஒரு கருவியைப் பயன்படுத்துகிறார்கள்.
 
-## IDE
+## ஐடிஇ(IDE)
 
-The term [IDE](https://en.wikipedia.org/wiki/Integrated_development_environment) (Integrated Development Environment) refers to a powerful editor with many features that usually operates on a "whole project." As the name suggests, it's not just an editor, but a full-scale "development environment."
+[ஐடிஇ] (https://en.wikipedia.org/wiki/Integrated_development_environment) (ஒருங்கிணைந்த மேம்பாட்டு சூழல்) என்பது "முழு திட்டப்பணியிலும்" பொதுவாக செயல்படும் பல பண்புகளைக் கொண்ட சக்திவாய்ந்த தொகுப்பியைக் குறிக்கிறது. அதன் பெயர் குறிப்பிடுவது போல, இது ஒரு தொகுப்பி மட்டுமல்ல, முழு அளவிலான "மேம்பாட்டு சூழல்".
 
-An IDE loads the project (which can be many files), allows navigation between files, provides autocompletion based on the whole project (not just the open file), and integrates with a version management system (like [git](https://git-scm.com/)), a testing environment, and other "project-level" stuff.
+ஐடிஇ ஒரு திட்டத்தை மின்னேற்றுகிறது (இது பல கோப்புகளை உள்ளடக்கியதாக இருக்கலாம்), கோப்புகளுக்கு இடையில் வழிசெலுத்தலை (navigation) அனுமதிக்கிறது, முழு திட்டத்தின் அடிப்படையில்  (திறந்த கோப்பு மட்டுமல்ல) தன்னியக்கத்தை வழங்குகிறது மற்றும் பதிப்புரு மேலாண்மை அமைப்பு([git](https://git-scm.com/) போன்றவை), சோதனை சூழல் மற்றும் பிற "திட்ட-நிலை" தேவைகளுடன் ஒருங்கிணைக்கிறது.
```

Comment:
`தன்னியக்கத்தை` seems to be a hard word. Can we add its English equivalent?

```suggestion
ஐடிஇ ஒரு திட்டத்தை மின்னேற்றுகிறது (இது பல கோப்புகளை உள்ளடக்கியதாக இருக்கலாம்), கோப்புகளுக்கு இடையில் வழிசெலுத்தலை (navigation) அனுமதிக்கிறது, முழு திட்டத்தின் அடிப்படையில்  (திறந்த கோப்பு மட்டுமல்ல) தன்னியக்கத்தை (autocompletion) வழங்குகிறது மற்றும் பதிப்புரு மேலாண்மை அமைப்பு([git](https://git-scm.com/) போன்றவை), சோதனை சூழல் மற்றும் பிற "திட்ட-நிலை" தேவைகளுடன் ஒருங்கிணைக்கிறது.
```

Reply:
Sure. It would be appreciated